src/identity/index.ts

```diff
@@ -1,10 +1,13 @@
-import { Transaction } from "@scure/btc-signer";
+import { Transaction } from "@scure/btc-signer/transaction.js";
 import { SignerSession } from "../tree/signingSession";
 
 export interface Identity {
+    signerSession(): SignerSession;
+    xOnlyPublicKey(): Promise<Uint8Array>;
+    compressedPublicKey(): Promise<Uint8Array>;
```

Comment:
```suggestion
    publicKey(): Promise<Uint8Array>; // returns the compressed 33 bytes
```

xOnly is a subset of the compressedPublicKey, so maybe worth to save 1 method to simplify implementations ?